openpype/hosts/nuke/api/lib.py

```diff
@@ -3320,15 +3346,35 @@ def get_viewer_config_from_string(input_string):
         display = split[0]
     elif "(" in viewer:
         pattern = r"([\w\d\s\.\-]+).*[(](.*)[)]"
-        result = re.findall(pattern, viewer)
+        result_ = re.findall(pattern, viewer)
         try:
-            result = result.pop()
-            display = str(result[1]).rstrip()
-            viewer = str(result[0]).rstrip()
+            result_ = result_.pop()
+            display = str(result_[1]).rstrip()
+            viewer = str(result_[0]).rstrip()
         except IndexError:
             raise IndexError((
                 "Viewer Input string is not correct. "
                 "Missing bracket! {}"
             ).format(input_string))
 
     return (display, viewer)
+
+
+def create_viewer_profile_string(viewer, display=None, path_like=False):
+    """Convert viewer and display to string
+
+    Args:
+        viewer (str): viewer name
+        display (Optional[str]): display name
+        path_like (Optional[bool]): if True, return path like string
+
+    Returns:
+        str: viewer config string
+    """
+    if display:
+        if path_like:
+            return "{}/{}".format(display, viewer)
+        else:
+            return "{} ({})".format(viewer, display)
+    else:
+        return viewer
```

Comment:
No else statements :)
```suggestion
    if not display:
        return viewer

    if path_like:
        return "{}/{}".format(display, viewer)

    return "{} ({})".format(viewer, display)
```